small-talk/pages/events/index.js

```diff
@@ -14,7 +14,55 @@ export const Events = ({data}) =>{
     return(
         <div className='bg-slate-800 text-white'>
             <Layout width={windowWidth} renderHeader={true} renderFooter={true}>
-                <h1>Events Page</h1>
+                <h1 className = 'font-bold'>Today&apos;s Events</h1>
+                <ul className = 'divide-y divide-solid'>
+                    <li className = 'list-disc'>
+                        <ul>
+                            <li>2024-5-2</li>
+                            <li>Toy Drive</li>
```

Comment:
Only criticism is there is no use of templates for the events. The events are "hard coded" and have the possibility of being of non-uniform structure. 

Maybe if there's time we can implement a list of uniform templates with unique event data down the line